pyreal/transformers/scale.py

```diff
@@ -0,0 +1,124 @@
+from sklearn.preprocessing import MinMaxScaler as SklearnMinMaxScaler
+from sklearn.preprocessing import Normalizer as SklearnNormalizer
+from sklearn.preprocessing import StandardScaler as SklearnStandardScaler
+
+from pyreal.transformers import Transformer
+from pyreal.transformers.wrappers import DataFrameWrapper
+
+
+class MinMaxScaler:
+    """
+    basically a sklearn MinMaxScaler but maintains DataFrame type
+    """
+
+    def __init__(self, feature_range=(0, 1), *, copy=True, clip=False):
+        """initialize a wrapped transformer and DataFrameWrapper, then wrap the DataFrameWrapper
+
+        Args:
+            feature_range (tuple (min, max), default=(0, 1)):
+                Desired range of transformed data.
+            copy (bool, default=True):
+                Set to False to perform inplace row normalization and avoid a copy
+                (if the input is already a numpy array).
+            clip (bool, default=False):
+                Set to True to clip transformed values of held-out data
+                to provided feature range.
+        """
+        self.data_frame_wrapper = DataFrameWrapper(
```

Comment:
I purposely avoided using wrapped_transformer because the DataFrameWrapper class has its own wrapped_transformer attribute (so I'd end up calling self.wrapped_transformer.wrapped_transformer, which is arguably bad for understanding).

Reply:
I don't think you ever need to access the DataFrameWrapper's wrapped_transformer - its functions will handle all the functionality you need.